vcpkg/maintainers/handling-usage-files.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Handling Usage Files
+description: Guidance for handling usage files.
+ms.date: 05/11/2023
+---
+
+# Handling Usage Files
+
+## Overview
+Usage files play an important role in providing instructions, examples, or guidelines for utilizing packages within our repository. To ensure consistency and clarity, we have established the following guidelines for handling usage files:
+
+### Using `configure_file()` Function
```

Comment:
`configure_file(... COPYONLY)`  is useful in project mode where it create file-level dependencies.
But why use `configure_file(... COPYONLY)` in script mode?

Alternatives:
- `file(COPY_FILE "a/usage" "b/usage")` - modern, but the target directory must exist. 
- `file(COPY "a/usage" DESTINATION "b")` - creates the directory if needed.
- `file(INSTALL "a/usage" DESTINATION "b")` - creates the directory if needed. Prints to stdout.

Reply:
`configure_file()` allows renaming and doesn't print to stdout.

Reply:
But there is nothing to rename and nothing to configure...

Reply:
`COPY_FILE` is already in use - in portfiles, CMake is recent enough.

Reply:
In example a rename is not required, however in other cases a rename may be required (providing different usage files based on feature selection). Unless it's excessively complicated, it's better to recommend one approach that works for all cases rather than "If you don't need to rename, do X, otherwise do Y."